### PR TITLE
modern CI setup from MQT template

### PR DESCRIPTION
**Pull request by lepistone**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1: if Travis doesn't complain

**Comment by guewen** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -1,39 +1,45 @@
+language: python
 sudo: false
-cache: pip
+cache:
+  apt: true
+  directories:
+    - $HOME/.cache/pip
+
+python:
+  - "2.7"
```
Comment (yvaucher) [inline .travis.yml:9]: Why not keeping the version just bellow the language ?

Comment (yvaucher) [inline .travis.yml:9]: Okay this is in the template. I guess we just want to change randomly the order sometimes...

**Comment by yvaucher** [comment]
Merging as it fixes travis

Thanks

**Pull request merged**